x/ecocredit/core/features/msg_bridge.feature

```diff
@@ -0,0 +1,182 @@
+Feature: MsgBridge
+
+  Scenario: a valid message
+    Given the message
+    """
+    {
+      "owner": "cosmos1depk54cuajgkzea6zpgkq36tnjwdzv4afc3d27",
```

Comment:
Are we able to use `regen` prefix'd bech32 addresses here instead of cosmos ones?

Reply:
It's initialized at runtime so I don't think we can within the unit tests but that would be nice. I took a look around and could not find a solution so I opened an issue to further investigate: #1243.

Reply:
i had this issue in the bridge repo, fixed it with this: https://github.com/rnd-inc/toucan-bridge-service/blob/ty/71-cosmos_client/regen/client.go#L21-L24

Reply:
Aha! Updated in the two unit tests here but we should update throughout in a separate pull request and move the config calls to the base suite of each package so that it applies to all unit tests. https://github.com/regen-network/regen-ledger/pull/1231/commits/974d53ef51f70d0f3d808660438e182f0b1c044e